### ISSUE TITLE
Height Is Broken on ``TextArea``

### ISSUE THREAD

**Issue by acederberg** [jlgraves-ubc/forms#8]
Thanks again for this extension!

The following qmd

```qmd
---
title: "Report an Issue"
form:
  action: POST ./
  method: GET
  submit: "Create Issue"
  id: create-issue
  fields:
    - label: Title
      name: title
      id: issue-title
      required: true
      type: text

    - label: Description
      name: description
      id: issue-description
      required: true
      type: textarea
      height: 256
---

If you found a bug, please fill out the form to submit an issue:

{{< form >}}
```

does not set the right height on ``TextArea``. This would appear to be from a broken joining:

```html annotate
<p>If you found a bug, please fill out the form to submit an issue:</p>
<p></p><div id="form-div" class="form-wrapper">
 <form id="form" action="POST ./" method="GET">
<div class="form-group">
<label for="issue-title" class="form-label mt-4">Title</label>
  <input type="text" id="issue-title" name="title" class="form-control" required="">
</div>
<div class="form-group">
<label for="issue-description">Description</label>
<textarea name="description" id="issue-description" rows="..&quot;256&quot;" cols="30" class="form-control"></textarea>
<!-- See 'rows' in the line above -->
</div>
<br>
<div class="form-group">
  <input type="submit" value="Create Issue" class="btn btn-primary">
</div>
</form></div>
<p></p>
```

**Comment by acederberg** [comment]
Found the bug, resolution should be included in #7 now.

**Comment by jlgraves-ubc** [comment]
Looks like a good fix!  Thanks.
